infra/tpu-pytorch-releases/artifacts.auto.tfvars

```diff
@@ -45,6 +50,25 @@ versioned_builds = [
     package_version = "1.13"
     accelerator     = "tpu"
   },
+  {
+    git_tag         = "v2.1.0"
+    package_version = "2.1"
+    accelerator     = "cuda"
+    cuda_version    = "11.8"
+  },
+  {
+    git_tag         = "v2.1.0"
+    package_version = "2.1"
+    accelerator     = "cuda"
+    cuda_version    = "11.8"
+    python_version  = "3.10"
+  },
+  {
+    git_tag         = "v2.1.0"
+    package_version = "2.1",
+    accelerator     = "cuda"
+    cuda_version    = "11.7"
+  },
```

Comment:
let's remove the 11.7 and do 12.0 instead

Reply:
Updated